pages/pipelines/environment_variables.md.erb

```diff
@@ -476,49 +476,52 @@ You can define environment variables in your jobs in a few ways, depending on th
 You can set environment variables in lots of different places, and which ones take precedence can get a little confusing.
 There are many different levels at which environment variables are merged together. The following walkthrough and examples demonstrate the order in which variables are combined, as if you had set variables in every available place.
 
-When a job runs on an agent, the first combination of environment variables happens in the job itself:
-
 ### Job environment
 
 When a job runs on an agent, the first combination of environment variables happens in the job environment itself. This is the environment you can see in a job’s Environment tab in the Buildkite dashboard, and the one returned by the REST and GraphQL APIs.
 
-The job environment is made by merging the following sets of values, in order:
+<div class="Docs__note">
+  <p>The precedence of environment variables is different from what's listed here if you are using <a href="https://buildkite.com/changelog/32-defining-pipeline-build-steps-with-yaml">YAML Pipeline Steps (beta)</a>.</p>
+  <p>For more information, please consult <a href="https://forum.buildkite.community/c/beta-support-feedback/yaml-pipeline-steps">our Community Forum post about the YAML Pipeline Steps beta</a>.</p>
```

Comment:
Should this link to: https://forum.buildkite.community/t/changes-to-environment-variable-precedence/107 ?

Reply:
Yes probably, this did work when I first copied it in but I think Discourse is giving me the runaround!